packages/block-library/src/comment-template/edit.js

```diff
@@ -122,9 +127,20 @@ export default function CommentTemplateEdit( {
 
 	const { rawComments, blocks } = useSelect(
 		( select ) => {
+			// Show empty comments if we don't have a postId context.
+			// The structure of the empty object as a rawComment allows
+			// to inner blocks to render the default placeholders.
+			if ( ! postId ) {
+				// We set a limit in order not to overload the editor of empty comments.
+				const defaultCommentsToShow = perPage <= 3 ? perPage : 1;
```

Comment:
Good question. My initial thought is to check `thread_comments_depth` and display enough placeholders to demonstrate maximum nesting.

Reply:
`thread_comments_depth` can go up to 10 though so that might be overkill 😅 

I've settled on a somewhat arbitrary maximum nesting of 3 for the placeholders. I think that should make the threading clear without overwhelming the UI.

Reply:
I don't know, I think that can be important. For example if you want to support a depth of 10, you may need to adjust the margins in order for that level of nesting to work. If you don't see a preview then you wouldn't be aware of such an issue.